### ISSUE TITLE
bug when launching in check-mode

### ISSUE THREAD

**Issue by copolycube** [bodsch/ansible-collection-prometheus#7]
https://github.com/bodsch/ansible-collection-prometheus/blob/f4917ec1860dc603a8e4f173d9352dc36c1a3bfc/roles/json_exporter/tasks/download.yml#L77C1-L83C62

when launching in check-mode, the prometheus role fails with the following error: 

```
TASK [bodsch.prometheus.json_exporter : define checksum for amd64 architecture] ***
Thursday 05 December 2024  13:35:24 +0000 (0:00:00.462)       0:00:12.047 ***** 
Thursday 05 December 2024  13:35:24 +0000 (0:00:00.463)       0:00:12.046 ***** 
fatal: [<obfuscated IP> -> localhost]: FAILED! => 
  msg: |-
    The conditional check '_latest_checksum.rc == 0' failed. The error was: error while evaluating conditional (_latest_checksum.rc == 0): 'dict object' has no attribute 'rc'
  
    The error appears to be in '<obfuscated path>/ansible/vendor-collections/ansible_collections/bodsch/prometheus/roles/json_exporter/tasks/download.yml': line 77, column 11, but may
    be elsewhere in the file depending on the exact syntax problem.
  
    The offending line appears to be:
  
  
            - name: define checksum for {{ system_architecture }} architecture
              ^ here
    We could be wrong, but this one looks like it might be an issue with
    missing quotes. Always quote template expression brackets when they
    start a value. For instance:
  
        with_items:
          - {{ foo }}
  
    Should be written as:
  
        with_items:
          - "{{ foo }}"
 ```

**Comment by bodsch** [comment]
Thank you!

I will have a look at this in the near future.

**Comment by bodsch** [comment]
Your issue should be fixed with Release 1.2.0
